integ-test/src/test/resources/correctness/expressions/text_functions.txt

```diff
@@ -1 +1,6 @@
-RIGHT('Hello World', 5) as column
\ No newline at end of file
+RIGHT('Hello World', 5) as column
+LEFT('Hello World', 5) as column
+ASCII('hello')
+LOCATE('world', 'helloworld')
+LOCATE('world', 'helloworld', 7)
+REPLACE('hello', 'world', 'opensearch')
```

Comment:
np: add another case such as REPLACE('hello world', 'world', 'search') ?

Reply:
Added the case